src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-zonegroup-form/rgw-multisite-zonegroup-form.component.ts

```diff
@@ -79,11 +97,17 @@ export class RgwMultisiteZonegroupFormComponent implements OnInit {
           }
         }),
         Validators.required
-      ])
+      ]),
+      placementTargets: this.formBuilder.array([])
     });
   }
 
   ngOnInit(): void {
+    _.forEach(this.multisiteZonegroupForm.get('placementTargets'), (placementTarget) => {
+      const fg = this.addPlacementTarget();
+      fg.patchValue(placementTarget);
+    });
```

Comment:
I don't know what patchValue does but you are patching the fg with the same value you are generating in the `addPlacementTarget()`. Probably the value should come from the input, hence the selection instead of the form, because the form should be filled and not used as a source of data.

Reply:
Am not completely sure what you mean here, if its related to the placement targets not getting added as you have shown in the screen recording you attached, probably you can try again

Reply:
I've tried again and it still fails. Nothing was changed in the code.

[screen-capture (2).webm](https://user-images.githubusercontent.com/30913090/229236811-6fe6d960-5fe0-4410-a84a-7095fb41bbbe.webm)